explainer/fenced_frame_config.md

```diff
@@ -0,0 +1,35 @@
+# Fenced frame configs
+
+
+# Introduction
```

Comment:
nit: I might make this a `##`, along with `Proposed solution`. And then `## Turtledove Example` would be `### Turtledove Example`.